### DISCUSSION TITLE
REST API one more level

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#8381] [Troubleshooting]
I'm trying to use the REST API and I need to go 1 more level deeper but It keep getting errors. I'm trying to get a sub category of pages in my marketplace but can only get all pages in the marketplace as I don't want to query that many pages.

**This works**
`http://localhost:3000/api/collections/marketplace/tree`

**This does not work**
`http://localhost:3000/api/collections/marketplace/{{ need_to_get_to_this_but_cant }}/tree`

**Comment by ryanmitchell** [comment]
I dont think this is possible with the REST Api tree.

You can filter on the entries end point:
http://localhost:3000/api/collections/marketplace/tree?filter[parent:is]={your_parent_id}&sort=order
